apps/console/src/features/data/country.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable object-curly-spacing */
```

Comment:
Shall we remove these comments disabling eslint? Check the other places as well.